ls1intum/Artemis
# `Development`: Move SAML2 docs to admin section

### Checklist
#### General



### Motivation and Context
The SAML2 setup is mostly relevant for administrators. It fits in in this documentation section with the ‘User Registration’.

### Description
Just moved the files to the updated location.

- Also fixes that the first image in the Cypress documentation was not shown due to the missing empty line between the `..figure` directive options and the caption. Removes trailing spaces in that file.
- In `programming-exercise-setup.inc` the indentation for a bullet point has been fixed.

### Steps for Testing
Look at the docs at https://artemis-platform--5016.org.readthedocs.build/en/5016/ and check that the SAML2 docs are present with all three images. Check that the Cypress documentation (Contributor Guide section) contains two images.

### Review Progress
#### Manual Tests
